tests/test_grpc_server.py

```diff
@@ -48,3 +49,10 @@ def test_batched_generation_request(grpc_client, grpc_server_thread_port):
 
     assert len(responses) == 2
     assert all(response.text for response in responses)
+
+
+@pytest.mark.skipif(vllm.config.is_cpu(), reason="Lora is not supported on cpu")
+def test_lora_request(grpc_client, lora_adapter_name):
+    response = grpc_client.make_request("hello", adapter_id=lora_adapter_name)
+
+    assert response.text
```

Comment:
@njhill just a simple sanity check for the grpc API, is there anything else you would like to test?